### ISSUE TITLE
Create content directory if it doesn't exist yet

### ISSUE THREAD

**Issue by dominikbraun** [verless/verless#201]
When creating a new file using `verless create file blog/coffee.md` and the `blog` directory doesn't exist inside `content`, verless will return an error.

Instead, we should check if `blog` exists inside `content` and create it if it doesn't. The area where this can be done is around here:

https://github.com/verless/verless/blob/3b809a29b22ffcb9c8d2f66595e942597c4bbac6/core/create.go#L140-L150

**Testing the issue**

You can test this issue by creating a file inside the included `example` project:

```shell script
$ go run cmd/verless/main.go create file -p example abc/my-file.md
```

This should work after fixing the issue, even if the `abc` directory doesn't exist inside `example/content` yet.

**Comment by GabrielFalcom** [comment]
I would like to contribute to this issue.

**Comment by dominikbraun** [comment]
Fixed with #208.
